morpheushelper/cogs/custom/__init__.py

```diff
@@ -0,0 +1,4 @@
+from . import translations as _
```

Comment:
do you need this import there?

Reply:
yes, this is needed to load the translation namespaces